pkg/webui/console/views/device-general-settings/utils.js

```diff
@@ -75,20 +76,17 @@ export const isDeviceABP = device =>
 export const isDeviceMulticast = device => Boolean(device) && Boolean(device.multicast)
 
 /**
- * Returns whether device keys are provisioned on external join server. Note: It is possible
- * to create end devices with root keys being provisioned by an external Join Server.
- * In this case both `root_keys.nwk_key` and `root_keys.app_key` must be missing in the payload.
+ * Returns whether an end device is not provisioned on the cluster join server.
```

Comment:
```suggestion
 * Returns whether an end device is provisioned on an external join server.
```
It is a bit confusing the way you worded it.